docs/helm-broker/03-04-bundles-docs.md

```diff
@@ -39,11 +39,4 @@ During the provisioning process, the Helm Broker pushes the **addonRepositoryURL
 
 ### Documentation structure
 
-Deliver documentation for your bundle in the Markdown files with the specified metadata. The metadata must contain the **title** and **type** fields:
-
-```
-title: Services and Plans
-type: Details
-```
-
-The **title** field defines the title of the document displayed in the Console.
+Deliver documentation for your bundle in Markdown files with specified metadata. To learn more about the metadata and content of the Markdown files with documentation, read [this](/components/headless-cms/#details-markdown-documents) document.
```

Comment:
```suggestion
Deliver documentation for your bundle in Markdown files with specified metadata. To learn more about the metadata and content of the Markdown files, read [this](/components/headless-cms/#details-markdown-documents) document.
```
I would remove the `with documentation` part as it is already suggested in the first sentence. :)